policy/modules/contrib/fdo.te

```diff
@@ -0,0 +1,56 @@
+policy_module(fdo, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+type fdo_t;
+type fdo_exec_t;
+init_daemon_domain(fdo_t, fdo_exec_t)
+
+type fdo_conf_t;
+files_config_file(fdo_conf_t)
+
+type fdo_unit_file_t;
+systemd_unit_file(fdo_unit_file_t)
+
+permissive fdo_t;
+
+########################################
+#
+# fdo local policy
+#
+allow fdo_t self:fifo_file rw_fifo_file_perms;
+allow fdo_t self:netlink_route_socket r_netlink_socket_perms;
+allow fdo_t self:tcp_socket create_stream_socket_perms;
+allow fdo_t self:udp_socket create_socket_perms;
+allow fdo_t self:unix_stream_socket create_stream_socket_perms;
+
+allow fdo_t fdo_exec_t:file execute_no_trans;
+
+manage_files_pattern(fdo_t, fdo_conf_t, fdo_conf_t)
```

Comment:
Add:
manage_lnk_files_pattern(fdo_t, fdo_conf_t, fdo_conf_t)

Reply:
It is strange that an app would modify its own config files?

Reply:
thank you for the point, I restricted access to fdo_conf_t. But fdo was creating files in /etc/fdo/aio/configs and /etc/fdo/aio/keys so I labeled them with rw suffix and allowed access.